src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/Interaction.vb

```diff
@@ -133,8 +134,13 @@ Namespace Microsoft.VisualBasic
             Return DirectCast(InvokeMethod("MsgBox", Prompt, Buttons, Title), MsgBoxResult)
         End Function
 
+        <UnconditionalSuppressMessage("ReflectionAnalsys", "IL2035:UnresolvedAssembly",
+                Justification:="This references Microsoft.VisualBasic.Forms assembly which is outside of the shared framework.")>
+        <UnconditionalSuppressMessage("ReflectionAnalsys", "IL2075:UnrecognizedReflectionPattern",
+                Justification:="DynamicDependency will preserve the required properties in order for the call to be safe.")>
+        <DynamicDependency(DynamicallyAccessedMemberTypes.PublicMethods, "Microsoft.VisualBasic._Interaction", "Microsoft.VisualBasic.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")>
```

Comment:
I'm not sure this is necessary. If the assembly is there, the linker should be able to tell it needs to preserve public methods on this type.

cc @vitek-karas